src/targets/gpu/kernels/include/migraphx/kernels/functional.hpp

```diff
@@ -288,6 +288,7 @@ constexpr auto join(G g, F f)
 template <class G, class F, class... Fs>
 constexpr auto join(G g, F f, Fs... fs)
 {
+    // return f1([=](auto x) { return f2([=](auto y) { return g(x, y); }); });
```

Comment:
Why is this here?

Reply:
To show how it works when there is only three arguments.